C#/C#.sublime-syntax

```diff
@@ -1498,15 +1514,15 @@ contexts:
 
   type_tuple:
     - match: \(
-      scope: punctuation.section.group.begin.cs
+      scope: punctuation.section.sequence.begin.cs
       set:
-        - meta_scope: meta.group.tuple.cs
+        - meta_scope: meta.sequence.tuple.cs
         - match: (\))
           captures:
-            1: punctuation.section.group.end.cs
+            1: punctuation.section.sequence.end.cs
```

Comment:
I guess the capture group could be removed?

Reply:
nice spot, done :)